ReSharper.FSharp/src/FSharp.Psi.Services/src/Generate/GenerateProvider.fs

```diff
@@ -336,6 +336,15 @@ type FSharpOverridingMembersBuilder() =
             objModelTypeRepr
 
         let (anchor: ITreeNode), indent =
+            let tryGetLastLet (typeMembersEnumerable: TreeNodeEnumerable<ITypeBodyMemberDeclaration>) =
+                let letBindings =
+                    typeMembersEnumerable
+                    |> Seq.takeWhile (fun x -> x :? ILetBindingsDeclaration)
+                    |> Seq.tryLast
+                match letBindings with
+                | Some b -> Some b
+                | None -> None
```

Comment:
What does this code actually try to achieve? 🙂

Reply:
Uh, just some leftover that should have been cleaned up.